### PR TITLE
[New Entry] JavaScript Strings: `.toString()`

### LINKED ISSUE #1027: [Term Entry] JavaScript Strings: toString()
### Reason/inspiration (optional)

Please write a new term entry under `content/javascript/concepts/strings/terms/` about the `.toString()` method of the `String` object in JavaScript. Let's include the following sections:

- A "Syntax" section with \```pseudo code snippet.
- An "Example" section with a \```js code snippet.

Bonus: If you're feeling extra adventurous, let's write an additional example in a "Codebyte Example" section that features a \```codebyte/javascript snippet.

To get started with the PR and entry file:

1. Under `content/javascript/concepts/strings/terms/`, create a new folder called `to-string`.
2. Then create a new file called `to-string.md` under `content/javascript/concepts/strings/terms/to-string/`.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by PriyanshuAhlawat**
closes #1027 

### Type of Change

- Created a new file `to-string.md` under `content/javascript/concepts/strings/terms/to-string/`

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (SSwiniarski) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:12]:
```suggestion
The **`.toString()`** method converts the given object and returns its corresponding string representation.
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (SSwiniarski) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:20]:
```suggestion
The `.toString()` method in JavaScript does not take any parameters.
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (SSwiniarski) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:21]:
```suggestion
It can be used with numbers, strings, arrays, and objects.
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (SSwiniarski) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:28]:
```suggestion
// Declaring variable with integer value
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (SSwiniarski) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:31]:
```suggestion
// Using toSting() method to get its string value
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (SSwiniarski) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:36]:
```suggestion
// Using typeof operator to show string value was returned
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@PriyanshuAhlawat, I did a first review and left you a few comments. For future entries, make sure your metadata is formatted correctly. You can look at the metadata section [here](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and look at the source of any content entry or term for a model.

**Comment by PriyanshuAhlawat** [comment]
@SSwiniarski thank you for the suggestions🙂, made all the suggested changes pls take a look

**Review by SSwiniarski — APPROVED** [review]
@PriyanshuAhlawat Looks good for second review.

**Comment by yangc95** [comment]
Hi @PriyanshuAhlawat 👋 I'll be taking this PR on for a second review.

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (yangc95) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:14]: Could we remove the empty line 13 or 14 so there is just one empty space between the ending dashes of the metadata and introduction?

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (yangc95) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:3]:
```suggestion
Description: 'Returns a string representing the object.'
```

Typo here: missing period

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (yangc95) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:21]:
```suggestion
## Syntax

```pseudo
stringObject.toString()
```
```

Removing the extra spacing in front for these lines

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (yangc95) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:23]:
```suggestion
The `.toString()` method does not take any parameters.
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (yangc95) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:24]: Can we turn this into a note like:

> **Note:** This method can be called on numbers, arrays, and objects.

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (yangc95) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:31]:
```suggestion
// Assigning constant to integer value
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (yangc95) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:34]:
```suggestion
// Using .toString() method to get its string value
```

**Review by yangc95 — COMMENTED** [review]
@PriyanshuAhlawat Finished conducting the second review. Please tag me once you've looked over the comments/suggestions or if you have any questions prior to making edits.

Thank you!

**Comment by PriyanshuAhlawat** [comment]
@yangc95 Thank you for the review. I have made the suggested changes, pls take a look🙂

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (yangc95) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:23]:
```suggestion
The `.toString()` method does not take any parameters.

> **Note:** This method can be called on numbers, arrays, and objects.
```

**Review by yangc95 — COMMENTED** [review]
@PriyanshuAhlawat This looks merge-ready! I just had one structural suggestion prior to merging. Please accept and we can get this added to JS Strings!

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@PriyanshuAhlawat I approved of this entry! Would you like to try resolving some of issues with the checks? Please see: [i-submitted-a-pr-but-tests-are-failing-how-do-i-fix-them](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md#i-submitted-a-pr-but-tests-are-failing-how-do-i-fix-them)

Also feel free to update your branch so this entry can be merged!

**Comment by PriyanshuAhlawat** [comment]
@yangc95 I ran yarn command to check for format issue, and it showed this
Screenshot (159)

**Comment by yangc95** [comment]
> @yangc95 I ran yarn command to check for format issue, and it showed this
> Screenshot (159)

@PriyanshuAhlawat Have you installed the packages prior to running those scripts?

**Comment by PriyanshuAhlawat** [comment]
@yangc95 can you mention the packages required

**Comment by yangc95** [comment]
> @yangc95 can you mention the packages required

I'm using `npm install` but it may be `yarn install` for you. If it states that you're missing any dependencies upon trying to install, follow the prompts. Lmk if you run into anything

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (Dusch4593) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:14]:
```suggestion
The **`.toString()`** method returns a string representation of a given value or object.
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (Dusch4593) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:22]:
```suggestion
The `.toString()` method takes no parameters.
```

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (Dusch4593) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:24]: What do we mean here? This entry is specifically about the `String.prototype.toString()` method. There are separate `.toString()` methods for numbers, arrays, and objects in the `Number`, `Array`, and `Objects` classes.

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (Dusch4593) [inline content/javascript/concepts/strings/terms/to-string/to-string.md:10]:
```suggestion
  - 'introduction-to-javascript'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@PriyanshuAhlawat Had a few comments/suggestions before we merge. 😄

File: content/javascript/concepts/strings/terms/to-string/to-string.md
Comment (PriyanshuAhlawat) [reply content/javascript/concepts/strings/terms/to-string/to-string.md:24]: Should I remove this note ?

**Review by Dusch4593 — APPROVED** [review]
@PriyanshuAhlawat LGTM! Renamed the folder and file in 9b9380a8df5cb3197c861a821cc3e8549b2fd2b0 (to `toString` and `toString.md`).

**Comment by Dusch4593** [comment]
Congrats on making a contribution to Codecademy Docs, @PriyanshuAhlawat! 🎉 🥳 🎊

The entry should now be live: https://www.codecademy.com/resources/docs/javascript/strings/toString

**Pull request merged**
